mailpoet/assets/js/src/wizard/steps/pitch_mss_step/first_part.tsx

```diff
@@ -49,7 +47,7 @@ function MSSStepFirstPart({
         <List>
           <li>{MailPoet.I18n.t('welcomeWizardMSSList1')}</li>
           <li>{MailPoet.I18n.t('welcomeWizardMSSList2')}</li>
-          {subscribersCount < 1000 ? (
+          {window.mailpoet_subscribers_count < 1000 ? (
```

Comment:
Please use `MailPoet.subscribersCount` we already have this one defined in the MailPoet module.

Reply:
See above: https://github.com/mailpoet/mailpoet/pull/4650#discussion_r1091977707